src/ice/candidate.rs

```diff
@@ -481,13 +487,6 @@ fn is_valid_ip(ip: IpAddr) -> bool {
     }
 }
 
-// TODO: maybe a bit strange this is used for SDP serializing?
```

Comment:
I moved this function closer to the struct, and removed the comment as it isn't being used by sdp now.